encryption/BIKE/BIKE/impl/ref.yaml

```diff
@@ -0,0 +1,5 @@
+name: Reference
+platform: c
+type: reference
+side channel guarding:
+  branching: true
```

Comment:
`timing: false` (§ 7.2)
> It is not designed to run in constant-time, as required for real-world implementation to offer side-channel resistance.

And I'd sasy branching is false, too. Or we can omit it, but where did you find something that suggests the `true` here?

Reply:
I am so sorry, I accidentally misread the phrase as "it is designed" instead of "is not designed". I will fix that too.